isis/src/lro/apps/lronaccal/lronaccal.xml

```diff
@@ -8,42 +8,67 @@
     </p>
     <p>
-    The LROC NAC camera has the ability to acquire images of differing sizes in
-    both line and sample.  The starting hardware detector pixel for the
-    acquired image is specified by the ISIS label keyword, SampleFirstPixel.
-    The first pixel in the detector is indicated by a value of 0.
+      The LROC NAC camera has the ability to acquire images of differing sizes in
+      both line and sample.  The starting hardware detector pixel for the
+      acquired image is specified by the ISIS label keyword, SampleFirstPixel.
+      The first pixel in the detector is indicated by a value of 0.
     </p>
 
     <p>
-    Dark current pixels are taken for each line from the masked pixels
-    that lie along each edge of the image.
+      Dark current pixels are taken for each line from the masked pixels
+      that lie along each edge of the image.
     </p>
     
     <p>
-    If SpatialSumming is 1 the dark current pixels are averaged together then
-    this average is subtracted from all image pixels.  If SpatialSumming is 2,
-    the dark current pixels for the A and B channel are averaged separately,
-    then the A channel dark average is subtracted from the A channel image
-    pixels and the B channel dark average is subtracted from the B channel
-    image pixels.
+      If SpatialSumming is 1 the dark current pixels are averaged together then
+      this average is subtracted from all image pixels.  If SpatialSumming is 2,
+      the dark current pixels for the A and B channel are averaged separately,
+      then the A channel dark average is subtracted from the A channel image
+      pixels and the B channel dark average is subtracted from the B channel
+      image pixels.
+    </p>
+
+    <p>
+      The DN level in an uncalibrated image is the sum of the true signal from the scene, 
+      the bias, the dark current, and random noise in all 3 components. The random noise in
+      the true signal and dark current is called shot noise and the random noise in the bias
+      is called read noise. The true signal, bias, and dark current are defined as mean
+      values so that if the random noise were averaged down to insignificance by taking a 
+      very large number of images and averaging them, the resulting image would be the true 
+      scene, bias, and dark current with no systematic error. That implies the statistical 
+      distribution of the random noise has an average of zero, and therefore the random noise
+      has both positive and negative values, except for the trivial case of zero random noise.
+    </p>
+
+    <p>
+      Let's look at the case of a calibrated image for which the true signal is zero, a dark 
+      image. In calibration, the mean bias and dark current are subtracted, but the random 
+      noise cannot be subtracted, because only the statistical distribution of the random 
+      noise can be known in advance, not the specific value for that pixel for that image. 
```

Comment:
`In calibration, the mean bias and dark current are subtracted, but the random 
       noise cannot be subtracted, because only the statistical distribution of the random 
       noise can be known in advance, not the specific value for that pixel for that image.` - This sentence still does not make sense.

Consider the following for the entire paragraph:
`Let's look at the case of a calibrated image for which the true signal is zero, a dark image. In calibration, the mean bias and dark current are subtracted. The random noise term is then randomly sampled from a known distribution with a mean of zero. Since the distribution has a mean of zero, values for the random noise can be positive or negative. Therefore, the addition of random noise to a pixel with true signal near zero can result in negative DN values.`

I would also note that this discussion would be significantly improved with the inclusion of a simple math equation. Something along the lines of:

reportedDN = ObservedDN - MeanBias - DarkCurrent + E, where E is a randomly sampled value from (mu, sigma^2) and mu=0.

Negative reported DNs are possible when E < ObserveredDN - MeanBias - DarkCurrent.